### ISSUE TITLE
Missing dependencies when managing config via $config_hash parameter

### ISSUE THREAD

**Issue by rauchrob** [rauchrob/puppet-kallithea#24]
Consider the manifest

``` puppet
class { kallithea:
  config_hash => { '[app:main]' => { 'host' => '0.0.0.0' } }
}
```

Then `kallithea::config` will call the `create_ini_settings` function _without_ the required dependency `Ini_setting[...] -> Exec['initialize kallithea config']`, i.e. it is possible that some of the `Ini_setting` resources get evaluated befor the `Exec`, leading to a bogus kallithea config.

**Comment by rauchrob** [comment]
Fixed in v0.4.1

**Comment by rauchrob** [comment]
Just to be clear about it: this bug could have happened only if you are using the `$config_hash` parameter when the Kallithea configuration file (defaults to `/srv/kallithea/kallithea.ini`) has not been created yet.
